libs/palladium/crates/atrium/authentik/.env.example

```
@@ -0,0 +1,18 @@
+# Copy to .env and replace both values with locally generated secrets.
+# Never commit .env or use these placeholders outside local development.
+PG_DB=authentik
+PG_USER=authentik
+PG_PASS=replace-with-a-local-postgres-password
+AUTHENTIK_SECRET_KEY=replace-with-a-long-random-secret-key
+
+# Loopback-only host ports.
+AUTHENTIK_HTTP_PORT=9000
+AUTHENTIK_HTTPS_PORT=9443
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTHENTIK_HTTPS_PORT key should go before the AUTHENTIK_HTTP_PORT key

(UnorderedKey)